contracts/test/MockUtilityToken.sol

```diff
@@ -0,0 +1,61 @@
+pragma solidity ^0.5.0;
+
+// Copyright 2017 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+// Auxiliary chain: MockUtilityToken
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+contract MockUtilityToken {
```

Comment:
This contract looks specific to a single unit test case. This can't be used for other unit test cases where one needs to assert balance of `beneficiary` and `facilitator`. Also, this can't be used for `redeem` process where we need methods like `approve` and `transferFrom`. I feel this contract should be more generic.